vendor/k8s.io/kubernetes/pkg/kubectl/cmd/util/factory_object_mapping.go

```diff
@@ -89,8 +89,7 @@ func (f *ring1Factory) objectLoader() (meta.RESTMapper, runtime.ObjectTyper, err
 		groupResources, err = discovery.GetAPIGroupResources(discoveryClient)
 	}
 	if err != nil {
-		glog.V(3).Infof("Unable to retrieve API resources, falling back to hardcoded types: %v", err)
-		return legacyscheme.Registry.RESTMapper(), legacyscheme.Scheme, nil
```

Comment:
@soltysh this breaks `--local` for `oc` commands

Reply:
Try figuring out why, it's always problematic when doing partial picks. Maybe you're missing something from upstream. Try debugging the path when `--local` is invoked in origin and see what might fix it.